customized_list_view/models/ir_module.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 ooops404
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models
+
+
+class Module(models.Model):
+    _inherit = "ir.module.module"
+
+    def _button_immediate_function(self, function):
+        res = super(Module, self)._button_immediate_function(function)
+        views_mods = self.env["custom.list.view"].search([])
+        for vm in views_mods:
```

Comment:
This solution is not optimal.
Please change this implementation without 2 cycles. If you want remove all 'custom.list.view.line' records without `field_id` value, you can use search for this. I think it's faster then you solution.